src/http2_connection.erl

```diff
@@ -444,7 +444,7 @@ route_frame({H, Payload},
                         0;
                     NewIWS ->
                         lager:debug("old IWS: ~p new IWS: ~p", [OldIWS, NewIWS]),
-                        OldIWS - NewIWS
+                        NewIWS - OldIWS
```

Comment:
For better or worse, it's a fix so I'm likely to include it here. I'll open an issue to deal with it.